src/Symfony/Component/HttpKernel/Tests/HttpCache/HttpCacheTest.php

```diff
@@ -1107,7 +1107,8 @@ public function testEsiCacheForceValidation()
         $this->assertEquals('Hello World! My name is Bobby.', $this->response->getContent());
         $this->assertNull($this->response->getTtl());
         $this->assertTrue($this->response->mustRevalidate());
-        $this->assertTrue($this->response->headers->hasCacheControlDirective('private'));
+        $this->assertFalse($this->response->headers->hasCacheControlDirective('private'));
```

Comment:
`private, no-cache` makes little sense. The Cache-Control header should only contain either `public`, `private` or `no-cache`.